src/may23/objects.js

```diff
@@ -0,0 +1,13 @@
+// For variable <obj1> define such pairs using dot syntax:
+// <hello: 'world'>, <ok: 'google'>, <best>: 777
+//
+// For variable <obj2> define the same pairs using array syntax
+
+const obj1 = {};
+const obj2 = {};
+
+// YOUR CODE:
+
+obj1.hello = 'world', obj1.ok = 'google', obj1.best = 777;
```

Comment:
yes, it is correct.
but please rewrite it with each statement on separate line style please.

BAD:
```
obj.hello = 'world', obj.ok = 'google';
```
GOOD:
```
obj.hello = 'world';
obj.ok = 'google';
```